### PR TITLE
Move all the autoyast profiles from Migration job groups to data/yam/

### PR DESCRIPTION
**Pull request by hjluo**
- Description
  - Move all the autoyast profiles from Migration job groups to data/yam/

Related ticket:
 - [131336](https://progress.opensuse.org/issues/131336)
- Related MR:
 - [320](https://gitlab.suse.de/coolgw/wegao-test/-/merge_requests/320)
 - [573](https://gitlab.suse.de/qa-maintenance/qam-openqa-yml/-/merge_requests/573)
Needles: N/A

Verification run:
[All_Verification_Runs](https://openqa.suse.de/tests/overview?arch=&flavor=&machine=&test=&modules=&module_re=&distri=sle&version=&build=hjluo%2Fos-autoinst-distri-opensuse%23new_mv_ay_profiles)

### CODE REVIEW

File: data/yam/autoyast/support_images/create_hdd_hpc_development_node_aarch64.xml.ep
Comment (lemon-suse) [inline data/yam/autoyast/support_images/create_hdd_hpc_development_node_aarch64.xml.ep:1]: For these hpc profiles, we can move them to support_images directly.

File: data/yam/autoyast/support_images/create_hdd_ha_x86_64.xml.ep
Comment (lemon-suse) [inline data/yam/autoyast/support_images/create_hdd_ha_x86_64.xml.ep:1]: For these ha profiles, we can move them to support_images directly.

File: data/yam/autoyast/support_images/create_hdd_sap.xml.ep
Comment (lemon-suse) [inline data/yam/autoyast/support_images/create_hdd_sap.xml.ep:1]: For these sap profiles, we can move them to support_images directly.

File: data/yam/autoyast/support_images/create_hdd_ha_x86_64.xml.ep
Comment (hjluo) [reply data/yam/autoyast/support_images/create_hdd_ha_x86_64.xml.ep:1]: Do you mean to move to `data/yam/support_images`? thanks

Comment (hjluo) [reply data/yam/autoyast/support_images/create_hdd_ha_x86_64.xml.ep:1]: done.

File: data/yam/autoyast/support_images/create_hdd_sap.xml.ep
Comment (hjluo) [reply data/yam/autoyast/support_images/create_hdd_sap.xml.ep:1]: done

File: data/yam/autoyast/support_images/create_hdd_hpc_development_node_aarch64.xml.ep
Comment (hjluo) [reply data/yam/autoyast/support_images/create_hdd_hpc_development_node_aarch64.xml.ep:1]: done.

**Review by rakoenig — APPROVED** [review]

**Pull request merged**